volume/driver/overlay_linux.go

```diff
@@ -2,16 +2,28 @@ package driver
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 	"syscall"
 )
 
 type OverlayDriver struct {
+	VolumesDir string
 	OverlaysDir string
 }
 
+func NewOverlayDriver(volumesDir, overlaysDir string) *OverlayDriver {
+	driver := &OverlayDriver{
+		VolumesDir:  volumesDir,
+		OverlaysDir: overlaysDir,
+	}
+	driver.RecoverMountTable(filepath.Join(volumesDir, "live"))
```

Comment:
Are we ignoring the error from `RecoverMountTable` on purpose here? 🤔

Reply:
nope, I missed it. thanks.